hack/common.sh

```diff
@@ -61,7 +61,7 @@ DEFAULT_KUSTOMIZE_BIN="${OUTDIR_BIN}/kustomize"
 
 DEFAULT_IMAGE_REGISTRY="docker.io"
 DEFAULT_REGISTRY_NAMESPACE="ibmcom"
-DEFAULT_IMAGE_TAG="latest"
+DEFAULT_IMAGE_TAG="0.2.0"
```

Comment:
Don't update the default tag here, suggest to restore it, thanks.

Reply:
latest tag is not accept by certification